### DISCUSSION TITLE
[FEATURE REQUEST] Breathing animation for RGB

### DISCUSSION THREAD

**Discussion by CorpseSea** [OpenStickCommunity/GP2040-CE#540] [General]
Is there any chance we can get this added in a future release? I feel theres a lot more potential for adding some nice customization with breathing LEDS; especially with the custom/gradient settings in the software!!

**Comment by bsstephan** [comment]
Please make a feature request under the GP2040-CE issues.

**Comment by TheTrainGoes** [comment]
Updated title
